modules/end-user-guide/partials/proc_mounting-a-secret-as-a-file-into-a-workspace-container.adoc

```diff
@@ -7,23 +7,57 @@
 
 WARNING: On 
 ifeval::["{project-context}" == "che"]
-{platforms-name} older than v1.13 and
+{kubernetes} older than v1.13 and
 endif::[]
-OpenShift {ocp3-ver}, secrets mounted as file overrides volume mounts defined in the devfile.
+OpenShift {ocp3-ver}, secrets mounted as a file override volume mounts defined in the devfile.
 
-This section describes how to mount a secret from the user's {orch-namespace} as a file in single-workspace or multiple-workspace containers of {prod-short}.
+In {prod-short}, you can mount the {platforms-name} secret from your {orch-namespace} as a file in single-workspace or multiple-workspace containers.
```

Comment:
What was wrong with the orig. version? In fact, I think it was more descriptive. The new formulation only vaguely suggests that something *can* be done.

Reply:
I already answered this question in the parallel comment in modules/end-user-guide/partials/proc_mounting-a-secret-as-an-environment-variable-into-a-workspace-container.adoc .